docs/samples/v1beta1/transformer/feast/README.md

```diff
@@ -0,0 +1,82 @@
+# Predict on an InferenceService with transformer using Feast online feature store 
+Transformer is an `InferenceService` component which does pre/post processing alongside with model inference. In this example, instead of typical input transformation of raw data to tensors, we demonstrate a use case of online feature augmentation as part of preprocessing. We use a [Feast](https://github.com/feast-dev/feast) `Transformer` to gather online features, run inference with a `SKLearn` predictor, and leave post processing as pass-through.
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+3. Your Feast online store is populated with driver [data](https://github.com/tedhtchang/populate_feast_online_store/blob/main/driver_stats.parquet), instructions available [here](https://github.com/tedhtchang/populate_feast_online_store), and network accessible.
+
+## Build Transformer image
+`KFServing.KFModel` base class mainly defines three handlers `preprocess`, `predict` and `postprocess`, these handlers are executed
+in sequence, the output of the `preprocess` is passed to `predict` as the input, when `predictor_host` is passed the `predict` handler by default makes a HTTP call to the predictor url 
+and gets back a response which then passes to `postproces` handler. KFServing automatically fills in the `predictor_host` for `Transformer` and handle the call to the `Predictor`, for gRPC
+predictor currently you would need to overwrite the `predict` handler to make the gRPC call.
+
+To implement a `Transformer` you can derive from the base `KFModel` class and then overwrite the `preprocess` and `postprocess` handler to have your own
+customized transformation logic.
+
+### Extend KFModel and implement pre/post processing functions
+We created a class, DriverTransformer, which extends KFModel for this driver ranking example. It takes additional arguments for the transformer to interact with Feast:
+* feast_serving_url: The Feast serving URL, in the form of `<host_name_or_ip:port>`
```

Comment:
How does user get the `feast_serving_url` ?

Reply:
The feast_serving_url is the endpoint for the Feast serving component. It could come from a standalone install or a Kubeflow based service with an accessible endpoint.

Reply:
The feast_serving_url is the feast-serving service name and port.  The command to get it is ` kubectl get svc -l app=feast-serving -n feast -o jsonpath='{.items[0].metadata.name}:{.items[0].spec.ports[?(@.name=="grpc")].port}'`. This should return something like `kf-feast-feast-serving:6566`

Reply:
In a standard Feast deployment on a kube cluster, the following script will return the feast_serving_url
`kubectl get svc -l app=feast-serving -n feast -o jsonpath='{.items[0].spec.clusterIP}:{.items[0].spec.ports[?(@.name=="grpc")].port}'`